qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -35,9 +35,9 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
     private static final String ASSETS_DIR = Paths.get(
             System.getProperty("user.dir"), "artifacts", "splunk-violations-test")
     private static final String PATH_TO_SPLUNK_TA_SPL = Paths.get(ASSETS_DIR,
-    "2021-07-23-TA-stackrox-1.2.0-input-validation-patched.spl")
+    "2023-06-16-TA-rhacs-2.0.0.spl")
     private static final String PATH_TO_CIM_TA_TGZ = Paths.get(ASSETS_DIR,
-    "splunk-common-information-model-cim_4190.tgz")
+    "splunk-common-information-model-cim_511.tgz")
```

Comment:
Why do you/we install CIM add-on as tarball and not as .spl?

Reply:
This is because the CIM is only available as a `.tgz` [in the store listing](https://classic.splunkbase.splunk.com/app/1621/).

Reply:
Okay. What may help prevent such question in the future is if you add a comment like
```java
// Downloaded from https://classic.splunkbase.splunk.com/app/1621/
```
above `PATH_TO_CIM_TA_TGZ` definition.

Reply:
Done!